ldk/goja/src/index.ts

```diff
@@ -0,0 +1,3 @@
+import * as clipboard from './clipboard';
+
+export { clipboard };
```

Comment:
TODO: Clipboard will need comments, and we need clipboard lock, listener cancellation which will be resolved in SIDE-2006.

Reply:
This was a placeholder, but now I'm thinking that we should expose an instance of the `ClipboardImpl` class.